system/api/server.go

```diff
@@ -9,4 +9,6 @@ func Run() {
 	http.HandleFunc("/api/content", Record(CORS(Gzip(contentHandler))))
 
 	http.HandleFunc("/api/content/external", Record(CORS(externalContentHandler)))
+
+	http.HandleFunc("/api/content/update", Record(CORS(updateContentHandler)))
```

Comment:
Just thinking ahead here.. do you think we should consider changing the design of the `/api` endpoints to simply respond to HTTP different request methods, and consolidate the endpoints to simply `/api/content/external`? 

This way, `externalContentHandler` would have a `switch` on `req.Method` and then we map: 

`POST` => `externalContentHandler` (which would be renamed to `addContentHandler`
`PUT` => `updateContentHandler` 

.. and so on as more are added. This way we wouldn't need to eventually add a `/api/content/delete`, just recognize a `DELETE` request method and add to the `switch`..

I suppose we'd then want to update the `api.Externalable` interface to be `api.Addable`, and refer to `Addable`, `Updateable`, (eventually) `Deleteable`, as `External` interfaces. 

If you don't think that this simplifies things, then we can leave as-is.. just want to provide Ponzu users with as easy of an experience as possible, and I personally think putting the control logic into the HTTP request method instead of the path is less for the programmer to remember or have to look up in docs. 

Would love to get your feedback on this.. 

Also, maybe get feedback from @olliephillips and @Sirikon? You guys have a good outlook on the project thus far.. maybe you'd like to help decide?